### ISSUE TITLE
RFID check-in/out (separate from the valid RFID tag/card functionality of the Admin portion).

### ISSUE THREAD

**Issue by Awmusic12635** [Awmusic12635/Wearables-Final-Group-Project#5]
a.	Using the Photon and RFID tags/cards, allow the checking in and checking out of a particular item.
b.	Communicate with the admin console and/or Firebase to check if the person is allowed to check out the device.
c.	Check to see if the device is available for checkout and only allow it to be checked out if it is.

**Comment by jrgabler** [comment]
Do we want to generate user activity on user details page?

**Comment by Awmusic12635** [comment]
Do you mean have an activity event if they modify their name or something like that?

**Comment by jrgabler** [comment]
I mean keep a history of what they checked out and attempts they made of they failed

**Comment by Awmusic12635** [comment]
Yes it should do that, it is listed under the admin console requirements
